torch/_torch_docs.py

```diff
@@ -8329,38 +8334,34 @@ def merge_dicts(*dicts):
                tensors for `U` and `Vh`, whereas :func:`torch.linalg.svd` returns
                empty tensors.
 
-.. note:: The singular values are returned in descending order. If :attr:`input` is a batch of matrices,
-          then the singular values of each matrix in the batch are returned in descending order.
-
 .. note:: The `S` tensor can only be used to compute gradients if :attr:`compute_uv` is `True`.
 
 .. note:: When :attr:`some` is `False`, the gradients on `U[..., :, min(m, n):]`
           and `V[..., :, min(m, n):]` will be ignored in the backward pass, as those vectors
           can be arbitrary bases of the corresponding subspaces.
 
-.. note:: The implementation of :func:`torch.linalg.svd` on CPU uses LAPACK's routine `?gesdd`
-          (a divide-and-conquer algorithm) instead of `?gesvd` for speed. Analogously,
+.. note:: The implementation of :func:`torch.linalg.svd` on CPU uses LAPACK's routine `gesdd`
+          (a divide-and-conquer algorithm) instead of `gesvd` for speed. Analogously,
           on GPU, it uses cuSOLVER's routines `gesvdj` and `gesvdjBatched` on CUDA 10.1.243
           and later, and MAGMA's routine `gesdd` on earlier versions of CUDA.
 
 .. note:: The returned `U` will not be contiguous. The matrix (or batch of matrices) will
           be represented as a column-major matrix (i.e. Fortran-contiguous).
 
-.. warning:: The gradients with respect to `U` and `V` will only be finite when the input does not
-             have zero nor repeated singular values.
-
-.. warning:: If the distance between any two singular values is close to zero, the gradients with respect to
-             `U` and `V` will be numerically unstable, as they depends on
-             :math:`\frac{1}{\min_{i \neq j} \sigma_i^2 - \sigma_j^2}`. The same happens when the matrix
-             has small singular values, as these gradients also depend on `S⁻¹`.
-
-.. warning:: For complex-valued :attr:`input` the singular value decomposition is not unique,
-             as `U` and `V` may be multiplied by an arbitrary phase factor :math:`e^{i \phi}` on every column.
+.. warning:: The singular values of a matrix are not unique. Any pair of left and right singular values may
```

Comment:
Should this be a warning or a note? Warning seems kind of extreme?

Reply:
This is interesting. 

We could, for example, return the singular values in descending order, and then the singular values would be unique, right? It's just that the columns of U (the left-singular vectors) and the columns of V (the right-singular vectors) are not unique. Is that what's meant by "left singular values" and "right singular values" above? I'm not familiar with that language.

Reply:
Yes. The singular values are always unique (same with eigenvalues for `eig` /  `eigh`). The singular vector / eigenvectors are what causes problems. I had corrected this errata in `torch.linalg.svd`, but I had forgotten to port it to `torch.svd`.

This now reads like this, tell me what you think:
![image](https://user-images.githubusercontent.com/3291265/115226755-826da980-a107-11eb-8b65-4089b68364c9.png)

About the warning vs note, I think that it's fine to leave it as a warning. It is surprisingly tricky to correctly use `U` and `Vh` in the loss function without them biting you due to the lack of unicity reasons. People expect functions to be continuous (and even differentiable) and `U` and `Vh` are not differentiable nor continuous, so I'd suggest to leave it as a warning.

Reply:
The note at the comment is excellent (I commented there).

OK, leaving it as a warning sounds OK.